README.rst

```diff
@@ -2,121 +2,79 @@
  OpenSlides
 ============
 
-I. What is OpenSlides?
-======================
+What is OpenSlides?
+===================
 
-OpenSlides is a free web-based presentation and assembly system for
-displaying and controlling of agenda, motions and elections of an assembly.
-See http://openslides.org/ for more information.
+OpenSlides is a free, web based presentation and assembly system for
+managing and projecting agenda, motions and elections of an assembly. See
+http://openslides.org/ for more information.
 
 
-II. Requirements
-================
+Requirements
+============
 
 OpenSlides runs everywhere where Python is running (for example on
 GNU/Linux, Mac or Windows (XP or newer)). On each client you need only a
 current version of a webbrowser.
 
 
-III. Installation
-=================
-
-Installation on GNU/Linux or Mac OS X
--------------------------------------
-
-1. Check requirements
-
-   Make sure that you have installed Python Programming Language 3 (>= 3.4)
-   on your system. You will also need the Python development headers.
-
-   For example for Ubuntu run::
-
-       $ sudo apt-get install python3-dev
-
-2. Setup a virtual Python environment using the virtual environment (venv)
-   package for Python (optional)
-
-   You can setup a virtual Python environment to install OpenSlides as
-   non-root user.
-
-   Note: For Ubuntu 14.04 you have to install the pyvenv binary package. Run::
-
-       $ sudo apt-get install python3.4-venv
-
-   Create your OpenSlides directory, change to it, setup and activate the
-   virtual environment::
-
-       $ mkdir OpenSlides
-       $ cd OpenSlides
-       $ python3 -m venv .virtualenv
-       $ source .virtualenv/bin/activate
-
-3. Install OpenSlides
-
-   You can use the package from the `OpenSlides Website
-   <http://openslides.org/download/>`_. Download latest OpenSlides release
-   as compressed tar archive and run::
+Installation
+============
 
-       $ pip install openslides-x.x.tar.gz
+1. Installation on GNU/Linux or Mac OS X
+----------------------------------------
 
-   OpenSlides will install all required Python packages (see
-   requirements_production.txt).
+a. Check requirements
+'''''''''''''''''''''
 
+Make sure that you have installed `Python <https://www.python.org/>`_ (>=
+3.4) on your system. You will also need the Python development headers (e.
+g. ``python3-dev`` for Ubuntu).
 
-Installation on Windows
------------------------
+TODO: Check Ubuntu 14.04 and extra packages like libjpeg ...
```

Comment:
Added libjpeg-dev and zlib1g-dev. The venv is probably not required because you can also use the old virtualenv command instead of the build-in venv module.